mobile/src/components/atoms/Select/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface Option {
```

Comment:
Czy value zawsze będzie stirngiem? jeśli nie to można zrobić value jako genetyka Option<T> { ...., value: T }